tests/test_pipelines.py

```diff
@@ -50,9 +47,23 @@ def func1(arg, config=None):
     return pipeline, config, tmp
 
 
-@pytest.mark.parametrize(
-    "Executor", [FunctionPipelineExecutor, PrefectPipelineExecutor, DaskPipelineExecutor],
+@pytest.fixture(
+    scope="session",
+    params=[
+        ("pangeo_forge_recipes.executors.dask", "DaskPipelineExecutor"),
+        ("pangeo_forge_recipes.executors.function", "FunctionPipelineExecutor"),
+        ("pangeo_forge_recipes.executors.prefect", "PrefectPipelineExecutor"),
+        ("pangeo_forge_recipes.executors.beam", "BeamPipelineExecutor"),
+    ],
 )
+def Executor(request):
+    try:
+        module = pytest.importorskip(request.param[0])
```

Comment:
Sorry to keep nit-picking this section. Last time, I promise...

```suggestion
        module = importlib.import_module(request.param[0])
```

It seems redundant to use `importorskip` and also `pytest.skip`. (This will require adding `importlib` to the imports above.

Reply:
No problem, this nit makes sense to me.